UseCases/Sources/InstrumentsAppTracingInstrpkg/SpansExampleInstrument/SpansExampleInstrument/SpansExampleInstrument.instrpkg

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<package>
+	<id>org.swift.server.tracing.example.SpansExampleInstrument</id>
+	<title>SpansExampleInstrument</title>
+	<owner>
+		<name>Konrad `ktoso` Malawski</name>
+	</owner>
+
+    <!-- Instruments Developer Help: https://help.apple.com/instruments/developer/mac/current/ -->
```

Comment:
see here for docs how this format works, and also WWDC sessions on custom instruments:

- https://developer.apple.com/wwdc18/410 
- https://developer.apple.com/wwdc19/421